core/scripts/lantern/collect/common.js

```diff
@@ -5,9 +5,10 @@
  */
 
 /** @typedef {{devtoolsLog?: string, lhr: string, trace: string}} Result */
-/** @typedef {{url: string, wpt: Result[], unthrottled: Result[]}} ResultsForUrl */
+/** @typedef {{url: string, wpt: Result|null, wptRetries: number, unthrottled: Result|null, unthrottledRetries: number, errors?: string[]}} ResultsForUrl */
 /** @typedef {Result & {metrics: LH.Artifacts.TimingSummary}} ResultWithMetrics */
-/** @typedef {{results: ResultsForUrl[], warnings: string[]}} Summary */
+/** @typedef {{results: ResultsForUrl[]}} Summary */
+/** @typedef {import('../run-on-all-assets.js').Golden} Golden */
```

Comment:
Reminder that the `Summary` and all types in `collect/` can change without implication for the lantern baseline testing code - as long as we continue to target the `Golden` output format.